src/objects/button-group/button-group.stories.mdx

```diff
@@ -0,0 +1,88 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+import demo from './demo/demo.twig';
+
+<Meta
+  title="Objects/Button Group"
+  argTypes={{
+    tight: {
+      type: { name: 'boolean' },
+      defaultValue: false,
+      description: 'Controls the tight-spacing modifier',
+    },
+    grow: {
+      type: { name: 'boolean' },
+      defaultValue: false,
+      description: 'Controls the button full width modifier',
+    },
+    count: {
+      type: { name: 'number' },
+      defaultValue: 2,
+      description: 'Number of buttons to render',
+      control: {
+        type: 'range',
+        min: 1,
+        max: 10,
+        step: 1,
+      },
+    },
+  }}
+/>
+
+# Button Group
+
+The Button Group layout object can be used for aligning a group of related buttons.
+
+## Default
+
+```php
+{% embed '@cloudfour/objects/button-group/button-group.twig' %}
+  {% block content %}
+    {% include '@cloudfour/components/button/button.twig' with { label: 'Button 1' } %}
+    {% include '@cloudfour/components/button/button.twig' with { label: 'Button 2' } %}
+  {% endblock %}
+{% endembed %}
+```
+
+<Canvas>
+  <Story name="Default">{(args) => demo({ ...args })}</Story>
+</Canvas>
+
+## Tight modifer
+
+You can apply a tighter-spacing via the template `tight` boolean argument:
+
+```php
+{% embed '@cloudfour/objects/button-group/button-group.twig' with { tight: true } %}
+  ...
+{% endembed %}
+```
+
+<Canvas>
+  <Story name="Tight" args={{ count: 5, tight: true }}>
+    {(args) => demo({ ...args })}
+  </Story>
+</Canvas>
+
+## Grow modifer
+
+You can have the buttons grow to fill their horizontal space via
+the template `grow` boolean argument:
+
+```php
+{% embed '@cloudfour/objects/button-group/button-group.twig' with { grow: true } %}
+  ...
+{% endembed %}
+```
+
+<Canvas>
+  <Story name="Grow" args={{ count: 5, grow: true }}>
+    {(args) => demo({ ...args })}
+  </Story>
+</Canvas>
+
+## Template Properties
+
+- `class`: Append a class to the root element.
+- `content` (block): Block to add Button elements.
+- `tight` (boolean, default: `false`): Adds the `o-button-group--tight` CSS class modifier.
+- `grow` (boolean, default: `false`): Adds the `o-button-group--grow` CSS class modifier.
```

Comment:
Thanks for documenting this!

Reply:
I was inspired by @tylersticka's previous work! 😄